app/uk/gov/hmrc/pensionschemereturnsipp/connectors/MinimalDetailsConnector.scala

```diff
@@ -55,6 +55,7 @@ class MinimalDetailsConnectorImpl @Inject() (appConfig: AppConfig, http: HttpCli
     http
       .get(url"$url")
       .setHeader(idType -> idValue, "loggedInAsPsa" -> loggedInAsPsa.toString)
+      .transform(_.withRequestTimeout(appConfig.ifsTimeout))
```

Comment:
Unless there is a specific reason, I don't think it is needed here, as this is a different service (not IF).